docs/api/ref/DesktopAgent.md

```diff
@@ -685,8 +665,80 @@ await fdc3.raiseIntentForContext(context, targetAppMetadata);
 ```
 
 #### See also
+
 * [`raiseIntent()`](#raiseintent)
 * [`Context`](Types#context)
-* [`TargetApp`](Types#targetapp)
+* [`AppIdentifier`](Metadata#AppIdentifier)
 * [`IntentResolution`](Metadata#intentresolution)
 * [`ResolveError`](Errors#resolveerror)
+
+## Deprecated Functions
+
+### `addContextListener` (deprecated)
+
+```ts
+addContextListener(handler: ContextHandler): Promise<Listener>;
+```
+
+Adds a listener for incoming context broadcasts from the Desktop Agent. Provided for backwards compatibility with versions FDC3 standard <2.0.
+
+#### See also
+* [`addContextListener`](#addcontextlistener)
+
+
+### `getSystemChannels` (deprecated)
+
+```ts
+getSystemChannels() : Promise<Array<Channel>>;
+```
+
+Alias to the [`getUserChannels`](#getuserchannels) function provided for backwards compatibility with version 1.1 & 1.2 of the FDC3 standard.
+#### See also
+* [`getUserChannels`](#getuserchannels)
+
+### `joinChannel` (deprecated)
+
+```ts
+joinChannel(channelId: string) : Promise<void>;
+```
+
+Alias to the [`joinUserChannel`](#joinuserchannel) function provided for backwards compatibility with version 1.1 & 1.2 of the FDC3 standard.
+
+#### See also
+* [`joinUserChannel`](#joinuserchannel)
+
+### `open` (deprecated)
+
+```ts
+open(name: String, context?: Context): Promise<AppMetadata>;
+```
+
+Version of `open` that launches an app by by name rather than `AppIdentifier`. Provided for backwards compatibility with versions FDC3 standard <2.0.
+
+#### See also
+
+* [`open`](#open)
+
+### `raiseIntent` (deprecated)
+
+```ts
+raiseIntent(intent: string, context: Context, name: String): Promise<IntentResolution>;
+```
+
+Version of `raiseIntent` that targets an app by by name rather than `AppIdentifier`. Provided for backwards compatibility with versions FDC3 standard <2.0.
```

Comment:
```suggestion
Version of `raiseIntent` that targets an app by name rather than `AppIdentifier`. Provided for backwards compatibility with versions FDC3 standard <2.0.
```